.pylintrc

```diff
@@ -0,0 +1,91 @@
+[MESSAGES CONTROL]
```

Comment:
You should also overwrite the default `MASTER` configuration:
```cfg
[MASTER]
jobs = 3
max-line-length = 99
```
Done just to speed up `pylint` and meet our 99-chars limit per line 😉

Reply:
There are also some additional extensions that we can/should use e.g.:
```cfg
[MASTER]
; http://pylint.pycqa.org/en/latest/technical_reference/extensions.html
load-plugins =
  pylint.extensions.bad_builtin,
  pylint.extensions.broad_try_clause,
  pylint.extensions.check_elif,
  pylint.extensions.comparetozero,
  pylint.extensions.empty_comment,
  pylint.extensions.emptystring,
  pylint.extensions.overlapping_exceptions,
```

Reply:
Did you find a moment to try to put it a more generic file? I would recommend `setup.cfg` as we can put there a configuration for the rest of the linters too 😉